index.html

```diff
@@ -0,0 +1,161 @@
+                    <img src="./img/pets-katrine.png" alt="katrine">
+                    <p>Katrine</p>
+                    <a href="">Learn more</a>
+                </div>
+                <div class="petcard">
+                    <img src="./img/pets-jennifer.png" alt="jennifer">
+                    <p>Jennifer</p>
+                    <a href="">Learn more</a>
+                </div>
+                <div class="petcard">
+                    <img src="./img/pets-woody.png" alt="woody">
+                    <p>Woody</p>
+                    <a href="">Learn more</a>
+                </div>
+            </div>
+            <button class="button d-block mx-auto mt-60 mb-100">Get to know the rest</button>
+        </div>
+        <div class="help">
+            <p class="block-title tac mx-auto">
+                How you can help our shelter
+            </p>
+            <div class="itemsfrow">
+                <div class="card">
+                    <img src="./img/icon-pet-food.png" alt="">
+                    <p>Pet food</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-transportation.png" alt="">
+                    <p>Transportation</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-toys.png" alt="">
+                    <p>Toys</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-bowls-and-cups.png" alt="">
+                    <p>Bowls and cups</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-shampoos.png" alt="">
+                    <p>Shampoos</p>
+                </div>
+            </div>
+            <div class="itemssrow">
+                <div class="card">
+                    <img src="./img/icon-vitamins.png" alt="">
+                    <p>Vitamins</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-medicines.png" alt="">
+                    <p>Medicines</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-collars-leashes.png" alt="">
+                    <p>Collars/leashes</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-sleeping-area.png" alt="">
+                    <p>Sleeping areas</p>
+                </div>
+            </div>
+        </div>
+        <div class="donation">
+            <div>
+                <img src="./img/donation-dog.png" alt="">
+            </div>
+            <div class="inform">
+                <p>You can also <br> make a donation</p>
+                <p class="bank">Name of the bank / Type of bank account</p>
+                <div class="number">
+                    <img src="./img/credit-card.png" alt="">
+                    <p class="num">8380 2880 8028 8791 7435</p>
```

Comment:
```suggestion
                    <a class="phone_number" href="tel:+8380 2880 8028 8791 7435">8380 2880 8028 8791 7435</a>
```
для телефонов пользуем ссылки
при клике - устройство попробует сделать вызов, что вроде как удобно :)

Так же название класса стоит заменить на более "говорящее"
не `num` a `phone_number` 

Учти, что меняя элемент, нам надо будет доработать стили (убрать синий цвет ссылки и подчеркивание как минимум)